src/main/java/com/bravo/user/model/filter/AddressFilter.java

```diff
@@ -0,0 +1,12 @@
+package com.bravo.user.model.filter;
+
+import java.util.Set;
+
+import lombok.Data;
+
+@Data
+public class AddressFilter {
```

Comment:
I would expect other commonly searchable fields like `cities` and `states` here.